### DISCUSSION TITLE
How can I pass data to button in a Table?

### DISCUSSION THREAD

**Discussion by Rigel772** [mantinedev/mantine#1988] [Q&A]
I would like to add a button to delete row in a table.
```
const rows = data.map((item) => (
    <tr key={item.id}>
      <td>{item.name}</td>
      <td>{item.price}</td>
      <td>
        <Button onClick={(item) => console.log("delete", item)}>
          <Trash />
        </Button>
      </td>
    </tr>
```

and `item` is `indefined`...
Any hint? Thanks

**Comment by rtivital** [comment]
Hi @Rigel772, your question is not related to Mantine. You should ask it in dedicated react community. You can find tables and other examples here – https://ui.mantine.dev
